src/components/App/App.s.ts

```diff
@@ -0,0 +1,30 @@
+import glamorous from 'glamorous';
+
+export const MyApp = glamorous.div({
+    backgroundColor: 'white'
+});
+
+export const Header = glamorous.div({
+    backgroundColor: 'rgb(34, 142, 230)',
+    padding: '10px',
+    color: 'white',
+    textAlign: 'center'
+});
+
+export const AppTitle = glamorous.div({
```

Comment:
tutaj może być Title, bo jeśli to jest w komponencie App to bedzie wiadomo że chodzi o AppTitle, a jeśli byśmy mieli jakiś zagnieżdzony Title w App, np w jakimś headerze ale on byłby w tym pliku to wtedy nazywamy go HeaderTitle